braid/braid_status.h

```diff
@@ -209,6 +219,29 @@ braid_StatusGetDone(braid_Status status,                   /**< structure contai
                     braid_Int   *done_ptr                  /**< output,  =1 if XBraid has finished, else =0 */
                     );
 
+/**
+ * Returns upper and lower time values on this processor. Two
+ * values are returned.
+ **/
+braid_Int
+braid_StatusGetTIUL(braid_Status status,                   /**< structure containing current simulation info */
+                    braid_Int   *iloc_upper,               /**< output, the upper time point index on this processor */
+                    braid_Int   *iloc_lower,               /**< output, the lower time point index on this processor */
+                    braid_Int   *level_ptr                 /**< input, level for the desired indices */
+                    );
+
+/**
```

Comment:
Is this comment correct?  I think GetTimeValues needs a different doc string here.  @jcc242

Reply:
After looking at these, I didn't like GetTIUL or GetTimeValues. Changed both in a7eb13f44271569788f222c1f12b16bca28e0002